src/__tests__/pdfUpload.integration.test.ts

```
@@ -0,0 +1,70 @@
+import request from 'supertest';
+import fs from 'fs';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)